modules/Forum/module.php

```diff
@@ -55,21 +55,11 @@ public function __construct(Language $language, Language $forum_language, Pages
         $pages->add('Forum', '/forum/view_topic', 'pages/forum/redirect.php');
         $pages->add('Forum', '/forum/view_forum', 'pages/forum/redirect.php');
 
-        // Hooks
-        EventHandler::registerEvent('newTopic',
-            $this->_forum_language->get('forum', 'new_topic_hook_info'),
-            [
-                'user_id' => $this->_language->get('admin', 'user_id'),
-                'username' => $this->_language->get('user', 'username'),
-                'nickname' => $this->_language->get('user', 'nickname'),
-                'content' => $this->_language->get('general', 'content'),
-                'content_full' => $this->_language->get('general', 'full_content'),
-                'avatar_url' => $this->_language->get('user', 'avatar'),
-                'title' => $this->_forum_language->get('forum', 'topic_title'),
-                'url' => $this->_language->get('general', 'url'),
-                'available_hooks' => $this->_forum_language->get('forum', 'available_hooks')
-            ]
-        );
+        EventHandler::registerListener(UserDeletedEvent::class, [DeleteUserForumHook::class, 'execute']);
+        EventHandler::registerListener(GroupClonedEvent::class, [CloneGroupForumHook::class, 'execute']);
+        EventHandler::registerListener(DiscordWebhookFormatterEvent::class, [ForumDiscordWebhookListener::class, 'execute']);
```

Comment:
Figure out a way for `ForumDiscordWebhookListener` to check the underlying event name that called the `DiscordWebhookFormatterEvent`